src/Data/Connection/AbstractConnectionResolver.php

```diff
@@ -332,7 +332,9 @@ public function setQueryArg( $key, $value ) {
 	 *
 	 * @return mixed|Model|null
 	 */
-	abstract public function get_node_by_id( $id );
+	public function get_node_by_id( $id ) {
+		return $this->loader->get( $id );
```

Comment:
Good call here. We should actually be able to do `$this->loader->load( $key );` and not add a new `get` method